plugins/attachyaml.yaml

```diff
@@ -1,9 +1,5 @@
-attachterraformyaml:
+attachyaml:
```

Comment:
attach is an out of date name (based on attaching internet gateways), is it worth renaming all of the workflows to aws (or awsinstance), aws2, etc for example?

Reply:
Something descriptive like "aws_vpc_ec2_sample" would be preferable.

Reply:
renamed